### PR TITLE
navbar floating on the logo bug fixed

### LINKED ISSUE #4117: Navbar item overflow on the logo
### Description

In the main page of layer 5 the nevbar item 'Solutions' overflow on the logo
https://layer5.io/
### Expected Behavior

It should not overflow in the top of the logo it should stay beside the logo
### Environment:
- Host OS: windows 10
- Browser:  Chrome

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). 
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by shibu-g**
**Description**

This PR fixes #4117

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
-[]Yes, I signed my commits.
  Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit b9c5285cf47db17c165f68f9900a8f04a9b24a82 at: https://6444bb502b67275bce1e90c5--layer5.netlify.app

**Comment by Shivam-AfA** [comment]
Thanks @shibu-g for the fix.

**Comment by Shivam-AfA** [comment]
You can correct the DCO error by going to the details link beside the failed DCO check.

**Comment by l5io** [comment]
🚀 Preview for commit 1ad78c363564c7c93dc1f687784585512c3c98bf at: https://6445142ef368c51741284b7e--layer5.netlify.app

**Review by thisiskaransgit — CHANGES_REQUESTED** [review]
There's already a media query at line 410, to avoid redundancy move the CSS, to it  
```javascript
    @media screen and (max-width: 992px) and (min-width: 912px) {
      width: 125px;
    }
``` 

And the toggle theme is still being cut-off, look into that too

**Comment by shibu-g** [comment]
> There's already a media query at line 410, to avoid redundancy move the CSS, to it  
> ```javascript
>     @media screen and (max-width: 992px) and (min-width: 912px) {
>       width: 125px;
>     }
> ``` 
> 
> And the toggle theme is still being cut-off, look into that too
> 
> 
> 

Yes thanks for pointing this out i will definitely  do it 👍

**Comment by l5io** [comment]
🚀 Preview for commit 84efe0fca44145624a235ae2494c9a7bb468b357 at: https://64460a8b788e667241de8e10--layer5.netlify.app

**Comment by Shivam-AfA** [comment]
Thanx @shibu-g for this.

**Comment by l5io** [comment]
🚀 Preview for commit 272014e29f80a2d00836a61a39e53d2cdea775e1 at: https://64461cc103dc5f7ae12df15b--layer5.netlify.app

**Comment by shibu-g** [comment]
> Thanx @shibu-g for this. Can you sign off your commits please?

Actually this time I did signoff on my commit.

**Comment by Shivam-AfA** [comment]
You need to sign off the previous commits as well

**Comment by shibu-g** [comment]
> You need to sign off the previous commits as well

i have tryed soo many method but not work 
i have try these method
  git rebase HEAD~4 --signoff
  git rebase -i HEAD~4
  git commit --amend --no-edit -s
nothing works
   how can i do this?

**Comment by Shivam-AfA** [comment]
@shibu-g 
Head over to this link and follow the steps: https://github.com/layer5io/layer5/pull/4121/checks?check_run_id=12964987112

**Comment by Chadha93** [comment]
@shibu-g Let's discuss this on the Websites call starting in 15 minutes. Please add this as an agenda, if you would. https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#

**Comment by leecalcote** [comment]
DCO manually cleared, @shibu-g 👍

File: src/sections/General/Navigation/navigation.style.js [hunk 1]
```diff
@@ -429,7 +429,15 @@ const NavigationWrap = styled.header`
       margin-top: -34px;
     }
   }
-
+  @media only screen and (min-width:912px) and (max-width:992px){
+    .nav{
+      margin-left: 22px;
```
Comment (leecalcote) [inline src/sections/General/Navigation/navigation.style.js:434]:
```suggestion
      margin-left: 1.375rem;
```

Prefer `rem`.

**Comment by l5io** [comment]
🚀 Preview for commit 488e5474a4fc6b2fe721c8d0c516b6332a91da38 at: https://644688acb39df43367cd57cf--layer5.netlify.app

**Pull request merged**